user/unified-platform.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Travis CI Single Platform"
+---
+
+On May 2nd, 2018 Travis CI integration open source and private projects onto a unified platform on **travis-ci.com**!
+
+This means you can manage public and private repositories from one domain, allow open source projects the ability to add extra concurrency, and allow our team to maintain and improve a single CI platform.
+
+We are implementing these changes with GitHub Apps, which not only provides a richer integration with GitHub, but also allows for many amazing features we plan to roll out in the next few months.
+
+## What This Means To You
+
+
+<!--  We should create a basic doc on our docs site which is mostly similar to a saved reply. It could/should explain that migration options will be available soon, you can auto opt into migration by setting up your open source repos on .com and we will email you when migration is available, or if you want to migrate now without retaining your build history, contact support? -->
+
+
+### If you're new to Travis CI
+
+If you are new to Travis CI, you can sign up at [travis-ci.com](https://www.travis-ci.com) for both open source and private repo testing. You will only need to provide your billing details if you want additional concurrency beyond our free open source support, or if you upgrade to a subscription after your 100-free-private-builds trial.
+
+### If you’re currently using travis-ci.org
+
+Travis CI will continue to support open source development as we've done in the past.  
+
+If you're an open source developer currently testing your project with Travis CI, your builds will continue to run on travis-ci.org for the moment. We will announce more soon on how to migrate your build history from travis-ci.org to travis-ci.com, but for now your currently active repositories will remain on the .org platform and you won’t notice any change at all.
+
+If you are activating a new repository on Travis CI, please use [travis-ci.com](https://www.travis-ci.com).
+
+#### Migrating to travis-ci.coming
+
+Sometime in May 2018 we'll be migrating repositories from travis-ci.org to travis-ci.coming
```

Comment:
I think this will happen in June? I'd be safer on that side and use June instead cc/ @joshk 

I think autocorrect may have slipped a typo on `travis-ci.coming` instead of `travis-ci.com` used both in the title and the text here, is that possible?